src/js/view/stage.js

```diff
@@ -1212,3 +1220,44 @@ silex.view.Stage.prototype.moveElements = function(elements, offsetX, offsetY) {
 silex.view.Stage.prototype.isMobileMode = function() {
   return goog.dom.classlist.contains(document.body, 'mobile-mode');
 };
+
+silex.view.Stage.prototype.handleRightClick = function(x, y) {
+
+  if(document.getElementById( 'powDiv' )!=null){
+    var el = document.getElementById( 'powDiv' );
+    el.parentNode.removeChild( el );  
+  }
+
+  var powDiv=goog.dom.createElement('div');
+  powDiv.id='powDiv';
+  
+  var leftPowDiv=goog.dom.createElement('div');
```

Comment:
this should be defined in the jade/pug/html templates, probably in `src/html/templates/workspace.jade`